source/accessories/amds/mainboard/index.md

```diff
@@ -120,7 +121,7 @@ The maximum supply current consumed by the IC including to drive currents for di
 
 A transformer is required for implementing isoSPI. This will provide an isolation barrier to the differential SPI signals. This is implemented by adding pulse transformer HX1188NLT, which has 1:1 turns ratio. More information on the pulse transformer is found in the [datasheet](https://media.digikey.com/pdf/Data%20Sheets/Pulse%20PDFs/10_100BASE-T%20Single%20Port%20SMD%20Magnetics_Rev2008.pdf).
 
-## Daugter Card Interface
+## Daughter Card Interface
```

Comment:
I'll do a Ctrl-F throughout the docs repo for daughtercard and replace with sensor card. I think that terminology works better since we've moved from "motherboard" to "mainboard".